docs/main/Solutioning/backup-and-restore/velero-br-fs.md

```diff
@@ -0,0 +1,316 @@
+    velero.io/csi-volumesnapshot-class: "true"
+driver: io.openebs.csi-mayastor
+deletionPolicy: Delete
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: mayastor-3-thin
+parameters:
+  protocol: nvmf
+  repl: "3"
+  thin: "true"
+provisioner: io.openebs.csi-mayastor
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+```
+
+In this example, We have deployed a sample Nginx test application for backup and restore
+
+**Command**
+
+```
+kubectl get pods -n mayastor-app
+```
+
+**Output**
+
+```
+NAME   READY   STATUS    RESTARTS   AGE
+test   1/1     Running   0          63m
+```
+
+**Sample Data**
+
+```
+kubectl exec -it  test -n mayastor-app -- bash
+root@test:/# cd /volume
+root@test:/volume# cat abc
+Mayastor velero Backup and restore
+root@test:/volume# exit
+exit
+```
+
+### Velero Backup and Restore
+
+- Use the following command to create a Velero backup:
+
+**Command**
+
+```
+velero backup create my-fs-backup --snapshot-volumes --include-namespaces=mayastor-app --volume-snapshot-locations=default --storage-location=default --snapshot-move-data
+```
+
+**Output**
+
+```
+Backup request "my-fs-backup" submitted successfully.
+```
+
+Run `velero backup describe my-fs-backup` or `velero backup logs my-fs-backup` for more details.
+
+- Use the following command to verify the backup status:
+
+**Command**
+
+```
+velero backup get
+```
+
+**Output**
+
+```
+NAME         STATUS     ERRORS  WARNINGS  CREATED                        EXPIRES  STORAGE LOCATION   SELECTOR
+my-fs-backup Completed  0       0         2024-07-30 08:48:12 +0000 UTC  29d      default            <none>
+```
+
+**Command**
+
+```
+kubectl get datauploads.velero.io -A
+```
+
+**Output**
+
+```
+NAMESPACE  NAME                STATUS     STARTED  BYTES DONE  TOTAL BYTES  STORAGE LOCATION  AGE    NODE
+velero     my-fs-backup-z9vrj  Completed  2m10s    35          35           default           2m10s  worker-velero-1
+```
+
+:::important
+In order to ensure that Velero can access the previously saved backups, it is recommended that you install Velero on the target cluster with the same values for the BUCKET-NAME and SECRET-FILENAME placeholders as you did originally. 
+:::
+
+- Use the following command to verify the backup on target cluster:
+
+**Command**
+
+```
+velero backup get
+```
+
+**Output**
+
+```
+NAME          STATUS      ERRORS   WARNINGS   CREATED                         EXPIRES   STORAGE LOCATION   SELECTOR
+my-fs-backup  Completed   0        0          2024-07-30 08:48:12 +0000 UTC   29d       default            <none>
+```
+
+- Use the following command to restore on target cluster:
+
+**Command**
+
+```
+velero restore create my-fs-restore --from-backup my-fs-backup --restore-volumes=true --namespace-mappings mayastor-app:mayastor-app-restore
+```
+
+**Output**
+
+```
+Restore request "my-fs-restore" submitted successfully.
+```
+
+Run `velero restore describe my-fs-restore` or `velero restore logs my-fs-restore` for more details.
```

Comment:
same here, this is also part of the output. when we submit the backup or restore, this logs and describe command pops up as part of the request.